plugins/services/src/js/structs/PodSpec.js

```diff
@@ -77,6 +78,9 @@ module.exports = class PodSpec extends ServiceSpec {
   getVolumes() {
     return this.get("volumes") || [];
   }
+  getVolumesData() {
+    return this.get("volumeData") || new VolumeList();
```

Comment:
not sure what's going on here... Is `volumeData` an instance of `VolumeList`?
I'd expect something like  `return new VolumeList({ items: this.get("volumeData") || [] });`

Reply:
You are right... But in this case, we don't need this function actually https://github.com/dcos/dcos-ui/pull/2937/files#diff-a70a8ec3027c4a35e120e723d1e44009R209

The functionality you are describing is already present on the pod the pod spec does not have this information I am going to remove this function.